python/DEV_SETUP.md

```diff
@@ -203,7 +197,7 @@ They should contain:
 - If necessary to further explain the logic a newline follows the first line and then the explanation is given.
 - The following three sections are optional, and if used should be separated by a single empty line.
 - Arguments are then specified after a header called `Args:`, with each argument being specified in the following format:
-    - `arg_name` (`arg_type`): Explanation of the argument, arg_type is optional, as long as you are consistent.
+    - `arg_name`: Explanation of the argument, arg_type is optional, as long as you are consistent.
```

Comment:
nit: if we are still mentioning arg_type in the comment, we perhaps don't want to remove it from the sample.